spec/spec_helper.rb

```diff
@@ -3,6 +3,7 @@
 require_relative '../lib/reek/spec'
 require_relative '../lib/reek/ast/ast_node_class_map'
 require_relative '../lib/reek/configuration/app_configuration'
+require 'active_support/core_ext/string/strip'
```

Comment:
The overall rationale here was that we use `strip_heredoc` in multiple places so it would make sense to require this in the spec_helper.
But you're right, this really doesn't belong to this PR, I just undid this change.